NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Ticket 40- Approve Posts

Description
Added ability for admins to approve posts. [Trello Ticket #40](https://trello.com/c/97ZFDxkU)

Type of Change
-Added new, non-breaking feature.

Testing Instructions


1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout sf-ticket-40
5. If you don't see the most recent version of my files, git pull origin sf-ticket-40
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in you client folder. Run "npm start" in your terminal.
11. Login as an author. [rdo1@timesonline.co.ukx](mailto:rdo1@timesonline.co.ukx) for the email parameter. Leave the password blank. Click "Login."
12. Click "Post Approval "
13. On any post, click approve.
14. When asked to confirm, click approve.
15. You should be redirected to the "All Posts" page and you should see the post you approved.



Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
